random_menu.rb

```diff
@@ -0,0 +1,19 @@
+# Random Menu Generator for Boozy Desserts (Lunch) - Loraine Kanervisto
+# Thanks Pinterest for the inspiration
+
+#Creates an adjective array
+adjs = ["salty", "buttery", "spicy", "creamy", "frosty", "bitter", "sour", "sweet", "chunky", "hot", "cheesy", "burnt", "candied", "drizzled", "fluffy"]
```

Comment:
Thanks! I keep forgetting to a literals, so fixing this gives me a good opportunity to practice.